### ISSUE TITLE
User fetching issue.

### ISSUE THREAD

**Issue by devr2k** [devr2k/mimic#3]
https://github.com/devr2k/mimic/blob/81a0b6af7389c18ff528f54a651e5f47842cf300/src/helpers/bot.js#L108

```
/home/devr2k/mimic/node_modules/discord.js/src/util/Constants.js:106
    if (userID.id) userID = userID.id;
               ^

TypeError: Cannot read property 'id' of undefined
```

**Comment by devr2k** [comment]
Issue found, handler.js expects the map to have a pure key, well a fresh install of this project does not provide one.

**Comment by devr2k** [comment]
Testing..

**Comment by devr2k** [comment]
Every command works. Issue solved as of 696bad5
 - [x] mimic member
 - [x] stop
 - [x] save & load
 - [x] victim
 - [x] mute & unmute
